src/controller/CHIPDeviceController.cpp

```diff
@@ -466,6 +469,13 @@ DeviceCommissioner::DeviceCommissioner() :
     mDeviceNOCChainCallback(OnDeviceNOCChainGeneration, this), mSetUpCodePairer(this)
 {}
 
+DeviceCommissioner::~DeviceCommissioner()
+{
+#if CHIP_DEVICE_CONFIG_ENABLE_WIFIPAF
+    DeviceLayer::ConnectivityMgr().WiFiPAFCancelConnect();
```

Comment:
How does this work with multiple connects from different commissioner instances that might be outstanding at the same time?

Reply:
https://github.com/project-chip/connectedhomeip/issues/34612